src/cryptoadvance/specter/api/rest/resource_jwt.py

```diff
@@ -0,0 +1,133 @@
+import jwt
+from flask import current_app as app
+from flask_restful import Api, abort, reqparse
+from cryptoadvance.specter.api.rest.base import (
+    BaseResource,
+    BasicAuthResource,
+    SecureResource,
+    rest_resource,
+    AdminResource,
+)
+import uuid
+import datetime
+import logging
+from ...user import *
+from .base import *
+
+from .. import auth
+
+logger = logging.getLogger(__name__)
+
+# Initialize the parser
+parser = reqparse.RequestParser()
+parser.add_argument(
+    "jwt_token_description", type=str, help="JWT token description", required=True
+)
+parser.add_argument("jwt_token_life", type=int, help="JWT token life", required=True)
+
+
+def generate_jwt(user, jwt_token_id, jwt_token_description, jwt_token_life):
+    # Generates a JWT token for the user
+    payload = {
+        "user": user.username,
+        "jwt_token_id": jwt_token_id,
+        "jwt_token_description": jwt_token_description,
+        "exp": datetime.datetime.utcnow() + datetime.timedelta(seconds=jwt_token_life),
+    }
+    return jwt.encode(payload, app.config["SECRET_KEY"], algorithm="HS256")
+
+
+def generate_token_id():
+    # Generates a unique token id
+    return str(uuid.uuid4())
+
+
+@rest_resource
+class ResourceJWT(BasicAuthResource):
```

Comment:
All the Resource-classes are called `SomethingResource` but this one is called `ResourceJWT`. Can you rename it to `JwtResource` ?